README.md

```diff
@@ -13,28 +13,29 @@
 ![GitHub license](https://img.shields.io/github/license/aissat/easy_localization?style=flat-square)
 
 <p align="center"> 
-Easy and Fast internationalizing your Flutter Apps,
-this package simplify the internationalizing process using Json file
+Easy and Fast internationalizing for your Flutter Apps,
+this package simplifies the internationalizing process using Json file
 </p>
 
 ## Why easy_localization
 
-- [x] simplifying and making easy the internationalizing process in Flutter.
-- [x] Using JSON, CSV, Yaml, Xml Files .
+- [x] simplifies and makes the internationalizing process in Flutter much easier.
+- [x] Uses easy_Localization_Loader JSON, CSV, Yaml, Xml Files .
```

Comment:
change to [Easy Localization Loader](https://github.com/aissat/easy_localization_loader)